examples/hello_world/my_script.py

```diff
@@ -41,4 +43,13 @@
     {"format": "png"},
     strict_path_visualization=False,
 )
-# dr.display_all_functions("./my_full_dag.dot", {"format": "png"})
+
+
+def custom_style(name: str, output_type: Type, tags: dict, node_type: str) -> dict:
+    print(name, output_type, tags, node_type)
+    if tags.get("some_key") == "some_value":
+        return {"fillcolor": "blue"}
+    return {}
```

Comment:
To hack in handling legend stuff:
```suggestion
def custom_style(name: str, output_type: Type, tags: dict, node_type: str) -> dict:
    print(name, output_type, tags, node_type)
    if node_type == "legend":
        return {"LLM": {"fillcolor": "blue"}, ...}
    else:
        if tags.get("some_key") == "some_value":
            return {"fillcolor": "blue"}
    return {}
```

Reply:
Or this outputs

```
{style}, name_of_style | None
```
We then use name_of_style for legend, if None, we don't include.

Reply:
Pass in the new `HamiltonNode` object -- has all this data + more.